industry/products.md

```diff
@@ -12,16 +12,19 @@ They vary by:
 - accuracy
 - price
 - customisation options
-- data confidentiality
-
+- [data confidentiality](data-confidentiality.md)
 and other features, as well as underlying architecture.
 
+Some providers offer ready-to-use solutions that can be integrated immediately.  These usually support many language pairs and their infrastructure is scalable.  An alternative is to train a model from scratch — or to use a pre-trained model — and deploy and maintain it.
 
-An advantage of using a machine translation provider is that the service is already working and can be integrated immediately. An alternative to using a machine translation provider is to train a model from scratch or to use a pre-trained model, and deploy and maintain a machine translation system.
-
-Unlike raw models, production machine translation systems usually provide language identification and sentence segmentation.
+Unlike raw models, ready-to-use solution usually include:
+- language identification
+- segmentation
+- bridging (pivoting)
```

Comment:
What would you call this?  I don't really like my wording, that's just what I call it.